docs/overrides/main.html

```diff
@@ -1,5 +1,9 @@
 {% extends "base.html" %}
 
+{% block announce %}
+🚨 RFC for <a href="https://github.com/aws-powertools/powertools-lambda-python/issues/4189">Powertools for AWS Lambda (Python) v3</a> is out!
```

Comment:
```suggestion
🚨 The next major version (v3) is coming - <a href="https://github.com/aws-powertools/powertools-lambda-python/issues/4189">come learn and discuss upcoming changes</a> is out!
```

reason: bring key info upfront, followed by call to action we want customers/community to do.

Reply:
Make much more sense! Changed!